server/stream.go

```diff
@@ -1352,7 +1386,7 @@ func (mset *stream) setupMirrorConsumer() error {
 	if ext != nil {
 		deliverSubject = strings.ReplaceAll(ext.DeliverPrefix+syncSubject(".M"), "..", ".")
 	} else {
-		deliverSubject = syncSubject("$JS.M")
+		deliverSubject = syncSubject("$JSDC.M")
```

Comment:
Is that permanent or what is part of some debug/test?

Reply:
Tracking GW sub leaks I thought it was better to not be under the $JS.> envelope since ephemerals go away when they detect interest goes away.

I could put it back.

Reply:
Just trying to reduce or at least be aware of all subjects that potentially need to be allowed/denied when it comes to permissions or import/export rules.

Reply:
Running a sub leak test now that R.I. showed me, so trying to reproduce that now.